<?xml version="1.0"?>
<launch>
    <arg name="camera_topic_name" default="/photoneo_center/sensor/camera_info"/>
    <arg name="image_topic_name" default="/photoneo_center/sensor/image_color"/>
    <arg name="inputcloud_topic_name" default="/cloud_without_segmented"/>
    <arg name="output_topic_name" default="/output"/>
    <arg name="target_frame" default="HV8"/>
    <arg name="source_frame" default="photoneo_center_optical_frame"/>
    <arg name="model_name" default="HV8"/>
    <arg name="world_frame" default="world"/>
    <arg name="f_scale" default="0.5"/>
    <arg name="cx_scale" default="0.99"/>
    <arg name="cy_scale" default="0.9"/>
    <arg name="radious" default="0.05"/>
    <arg name="dulation" default="0.1"/>
    <arg name="oculuder_topic_name" default="/tsuchida_object_occuluder"/>
    <arg name="timespan" default="4"/>
    
    
    
    
    
    
    
    <node name="colored_point" pkg="w2d_to_3d_ros" type="ano_and_exec" output="screen">
        <param name="camera_topic_name" value="$(arg camera_topic_name)"/>
        <param name="image_topic_name" value="$(arg image_topic_name)"/>
        <param name="inputcloud_topic_name" value="$(arg inputcloud_topic_name)"/>
        <param name="output_topic_name" value="$(arg output_topic_name)"/>
        <param name="target_frame" value="$(arg target_frame)"/>
        <param name="source_frame" value="$(arg source_frame)"/>
        <param name="model_name" value="$(arg model_name)"/>
        <param name="world_frame" value="$(arg world_frame)"/>
        <param name="f_scale" value="$(arg f_scale)"/>
        <param name="cx_scale" value="$(arg cx_scale)"/>
        <param name="cy_scale" value="$(arg cy_scale)"/>
        <param name="radious" value="$(arg radious)"/>
        <param name="dulation" value="$(arg dulation)"/>
        <param name="oculuder_topic_name" value="$(arg oculuder_topic_name)"/>
        <param name="timespan" value="$(arg timespan)"/>
    </node>
</launch>